automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/apps/AppLifecycleSpec.scala

```diff
@@ -53,12 +53,12 @@ class AppLifecycleSpec
     ("create CROMWELL app, delete it and re-create it with same disk",
      createAppRequest(AppType.Cromwell, "cromwell-test-workspace", None),
      false,
-     true
+     false
```

Comment:
Disabling testing PD reattachment for Cromwell because I don't think we support it anyway.